src/pseudopeople/configuration/validator.py

```diff
@@ -39,8 +46,18 @@ def validate_user_configuration(user_config: Dict, default_config: ConfigTree) -
                 default_noise_type_config = _get_default_config_node(
                     default_column_config, noise_type, "noise type", dataset, column
                 )
+                parameter_config_validator_map = {
+                    NOISE_TYPES.nickname.name: {
+                        Keys.CELL_PROBABILITY: _validate_nickname_probability
+                    }
+                }.get(noise_type, DEFAULT_PARAMETER_CONFIG_VALIDATOR_MAP)
```

Comment:
I don't totally understand why noise types can't just live inside the DEFAULT one?

Reply:
I thought I had a reason that I cannot remember but I update and this works so good call.